examples/basic/text/script.js

```diff
@@ -0,0 +1,30 @@
+import * as UTILS from '../../globals';
+
+const controlsModule = new WHS.OrbitControlsModule();
+
+const app = new WHS.App([
+  ...UTILS.appModules({
+    position: new THREE.Vector3(0, 10, 200)
+  }),
+
+  controlsModule
+]);
+
+const text = new WHS.Text({
+  text: 'Gentilis Bold Text',
+
+  parameters: {
+      font: `${process.assetsPath}/fonts/gentilis_bold.typeface.json`,
```

Comment:
@hirako2000 if `font` - then returns a `Mesh`, if `url` - then a `Promise` that resolves with a Mesh